tests/storage-disks-vm

```diff
@@ -109,6 +109,18 @@ waitInstanceReady v1
 lxc exec v1 -- ls -l /mnt/foo1
 lxc stop -f v1
 
+# Check adding a disk to a vm with a long name to test possible problems with long socket paths or long qemu device tags
+LONG_DEVICE_NAME="device-with-very-long-name-for-long-qemu-property-handling-tests"
+
+lxc init "${IMAGE}" v2 --vm
+lxc config device add v2 "${LONG_DEVICE_NAME}" disk source="${testRoot}/allowed1" path="/mnt/foo"
+lxc start v2
+waitInstanceReady v2
```

Comment:
I'd check if the cold-plugged device was mounted. I'd suggest using `lxc exec ... mountpoint /mnt/foo`